.scrutinizer.yml

```diff
@@ -1,4 +1,9 @@
 build:
+    dependencies:
+        before:
+            - composer require --dev spryker/console
+            - composer require --dev spryker/transfer
```

Comment:
Why not putting those in composer json?

Reply:
Since we need this for every repository, and only for Scrutinizer builds.

Reply:
transfer module - right, may be included in `composer.json`